Automattic/wc-calypso-bridge
# Add plan picker banner for free trial

Fixes https://github.com/Automattic/wc-calypso-bridge/issues/911

### Changes

1. Added free trial plan check to load frontend class conditionally.  The check logic was suggested in p1674072849311499-slack-C0Q664T29

2. Added a plan picker banner at the bottom.

### Test Instructions

1. Make sure to copy over the latest changes in [woocmmerce-calypso-bridge-helper.php](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178) and the plugin is activated.
2. Clone this branch
3. Navigate to the frontend of your local env. Confirm the banner is not rendered.
4. Change [this line](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178#file-a-woocommerce-calypso-bridge-helper-php-L33) to false in your local env.
5. Go back to the frontend of your local env and confirm the banner is rendered.
